tests/subyt/test_source_config.py

```
@@ -0,0 +1,111 @@
+    assert source is not None
+    assert isinstance(source, CSVFileSource)
+    with source as items:
+        data = list(items)
+        assert len(data) == 3
+        assert data[0] == {"name": "Bram", "age": "40"}
+        assert data[1] == {"name": "Steve", "age": "48"}
+        assert data[2] == {"name": "Marc", "age": "54"}
+
+
+def verify_output(outfile: Path) -> None:
+    with open(outfile, "r") as f:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)